### PR TITLE
Correct typos for posts.md

### PR DESCRIPTION
**Pull request by archersmind**

### CODE REVIEW

File: site/_docs/posts.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Jekyll 会自动取每篇文章从开头到第一次出现`excerpt_separator`的
 由于 Jekyll 会提取第一段的内容，你没有必要将摘要包裹在`p`标签中，它已经为你做了这项工作。如果你希望移除它们可以使用如下的代码：
 
 {% highlight html %}
-{{ post.excerpt | remove: '<p>' | remove: '</p>' }}
+{ post.excerpt | remove: '<p>' | remove: '</p>' }
```
Comment (xcatliu) [inline site/_docs/posts.md]: 应该是双大括号才对
https://raw.githubusercontent.com/jekyll/jekyll/master/site/_docs/posts.md

**Comment by archersmind** [comment]
updated

**Comment by xcatliu** [comment]
Thanks~

**Comment by xcatliu** [comment]
Published

**Pull request merged**